.travis.yml

```diff
@@ -17,7 +17,7 @@ cache:
   - directories: data
 install:
   - pip install docutils sphinx
-  - pip install -e .[test,louvain,leiden]
+  - pip install -e .[test,louvain,leiden,magic]
```

Comment:
@falexwolf @ivirshup how should we go about testing external stuff?

I’d think that we should probably test it in a separate job, i.e.

- python current - 1
- python current
- static analysis only
- extras only

Reply:
External job sounds like a good option. Only downside I immediately see (other than complexity) is that it will make CI take longer to run, but I don't think that's so bad.

I'm thinking we would control this with a new pytest tag, like we do the "internet test" ones?

Reply:
Yeah. Also not super urgent, only once there’s enough tested externals that it makes a difference.